docs/configuration.rst

```diff
@@ -130,6 +130,13 @@ Top level keys
 
     ``true`` by default.
 
+``build_ignore_filenames``
```

Comment:
Just a minor suggestion.

Since this is used by both `build` and `check` maybe keep the config name short.

```suggestion
``ignore_filenames``
```

Reply:
I'm changing it to just `ignore`, but if `ignore_filenames` or something else is preferable, let me know

Reply:
We can go witn `ignore` . Thanks. It was just a minor feedback.